presto-docs/src/main/sphinx/sql/insert.rst

```diff
@@ -7,18 +7,19 @@ Synopsis
 
 .. code-block:: none
 
-    INSERT INTO table_name query
+    INSERT INTO table_name [ ( column [, ... ] )  query
 
 Description
 -----------
 
 Insert new rows into a table.
 
-.. note::
```

Comment:
Thanks. It sounds much better when it is written by native speaker. Next time I will try harder to write something better.